experiments/bert_finetune/bert_glue_finetune.sh

```diff
@@ -44,7 +44,7 @@ GLUE_TASKS=(
 )
 
 DATA_DIR="${PWD}/glue-test-data"
-EPOCHS=2
+EPOCHS=1
```

Comment:
Is this change necessary? Is this to reflect the experimental setup? I wouldn't bother pushing this and instead write down our experiment settings in some file.

Reply:
Not really, I can change this back.